fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/digitalcash/Output.java

```diff
@@ -0,0 +1,49 @@
+package com.github.dedis.popstellar.model.network.method.message.data.digitalcash;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Objects;
+
+// Object representing an output of this transaction
+public class Output {
+  @SerializedName("value")
+  private final int value; // the value of the output transaction, expressed in miniLAOs
+
+  @SerializedName("script")
+  private final Script_output script; // The script describing the TxOut unlock mechanism
+
+  /**
+   * @param value the value of the output transaction, expressed in miniLAOs
+   * @param script The script describing the TxOut unlock mechanism
+   */
+  public Output(int value, Script_output script) {
+    this.value = value;
+    this.script = script;
+  }
+
+  public int get_value() {
```

Comment:
```suggestion
  private final long value; // the value of the output transaction, expressed in miniLAOs

  @SerializedName("script")
  private final Script_output script; // The script describing the TxOut unlock mechanism

  /**
   * @param value the value of the output transaction, expressed in miniLAOs
   * @param script The script describing the TxOut unlock mechanism
   */
  public Output(long value, Script_output script) {
    this.value = value;
    this.script = script;
  }

  public long get_value() {
```
Since we have to support amounts bigger than `2^31`